updating/index.adoc

```diff
@@ -0,0 +1,90 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. Platform administrators are automatically notified when an update is available for their cluster. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
+== Understanding the OpenShift Update Service
+xref:../updating/understanding-the-update-service.adoc#understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, check the following information:
+
+* xref:../updating/understanding-the-update-service.html#unmanaged-operators_understanding-the-update-service[Understanding support policy for unmanaged Operators]
```

Comment:
The page is not available.

Reply:
Addressed

Reply:
I could not find the page here. And I'm not sure the overview should refer to which version's doc. Take v4.9 for example, the `Understanding the OpenShift Update Service` was moved to https://docs.openshift.com/container-platform/4.9/updating/updating-restricted-network-cluster.html#update-restricted-network-cluster-update-service and there is not `Understanding support policy for unmanaged Operators` now. Could you confirm we should refer to which doc to check if the overview is complete or not?

Reply:
@jiajliu I confirmed on the channel. I needed to create 2 separate PRs for different versions. For eg: there are 2 sections that don't belong to 4.6 and 4.7 so I created separate PRs for them. Hopefully, that will resolve the `page not found` error. Would you please review those PRs? 
[1] For 4.7 ==> https://github.com/openshift/openshift-docs/pull/43644
[2] For 4.8+ ==> https://github.com/openshift/openshift-docs/pull/43645

Reply:
I've already added comments in above prs. BTW, v4.8 looks like a little different from v4.9+. We may need seperate PRs for them too.